virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc

```diff
@@ -12,6 +12,11 @@ You do not need to perform any action to enable smart-cloning, but you need to e
 
 When you create a data volume with a persistent volume claim (PVC) source, you automatically initiate the cloning process. You always receive a clone of the data volume, if your environment supports smart-cloning or not. However, you will only receive the performance benefits of smart cloning if you storage provider supports smart-cloning.
 
+[TIP]
+====
+When you globally enable preallocation, the Containerized Data Importer preallocates disk space to improve write performance during cloning. See xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Using preallocation for data volumes] for more information.
```

Comment:
Preallocation does not happen when smart cloning is used.  Instead, the storage provides the clone and the preallocation state of the clone is storage dependent.

Reply:
Thanks, will remove this Tip from the smart cloning assembly.